src/test/run-pass/i128.rs

```diff
@@ -109,4 +109,8 @@ fn main() {
     assert_eq!(b(-87559967289969187895646876466835277875_i128) /
                b(84285771033834995895337664386045050880_i128),
                -1i128);
+
+    // iter-arithmetic traits
+    assert_eq!(10, [1, 2, 3, 4].iter().sum());
```

Comment:
This presumably isn't testing the `i128` implementation. It will probably need to be something like `assert_eq!(10i128, [1i128, 2, 3, 4].iter().sum());`.